larray/tests/test_array.py

```diff
@@ -88,6 +88,13 @@ def setUp(self):
         self.small_data = np.arange(30).reshape(2, 15)
         self.small = LArray(self.small_data, axes=(self.sex, self.lipro), title=self.small_title)
 
+    @pytest.fixture(autouse=True)
+    def setup(self, tmpdir):
+        self.tmpdir = tmpdir.strpath
+
+    def get_path(self, fname):
```

Comment:
rename get_path to tmp_path or get_tmp_path? Currently, when I see get_path and inputpath, it is not obvious from their name to know which does what, which needs to be used.